docs/api/sql/Raster-aggregate-function.md

```diff
@@ -1,9 +1,9 @@
 ## RS_Union_Aggr
 
-Introduction: Returns a raster containing bands by specified indexes from all rasters in the provided column. Extracts the first bands from each raster and combines them into the output raster based on the input index values.
+Introduction: This function combines multiple rasters into a single multiband raster by stacking the bands of each input raster sequentially. The function arranges the bands in the output raster according to the order specified by the index column in the input. It is typically used in scenarios where rasters are grouped by certain criteria (e.g., time and/or location) and an aggregated raster output is desired.
```

Comment:
Does this change the behavior of the old function? The old function only takes the first band of all input raster. Now it takes all bands of each raster? @prantogg

Reply:
Yes, it essentially stacks all bands of each raster in order of `index`.

Input dataframe - 
```
idx |   Raster
--------------------------------
1    |   raster1 (R1 | G1 | B1)
2    |   raster2 (R2 | G2 | B2)
3    |   raster3 (R3 | G3 | B3)
```

```
RS_Union_aggr(Raster, idx)
```

resultant raster -

`raster (R1 | G1 | B1 | R2 | G2 | B2 | R3 | G3 | B3 )`